app/src/utils/snowbridge.ts

```
@@ -79,3 +77,74 @@ export async function trackFromEthTx(
   )
   return transfers
 }
+
+export async function trackFromAhToEthTx(
+  env: environment.SnowbridgeEnvironment,
+  skipLightClientUpdates = SKIP_LIGHT_CLIENT_UPDATES,
+  historyInSeconds = HISTORY_IN_SECONDS,
+): Promise<FromAhToEthTrackingResult[]> {
+  if (!env.config.SUBSCAN_API) {
+    console.warn(`No subscan api urls configured for ${env.name}`)
```

Should we return an error? I'd say this is important to track with Sentry

---

We could indeed. I think the API route will notify sentry and return an empty array. 

Let me add this!